Zadanie 17.4 Praktyczne wykorzystanie szablonów/views/log-page.pug

```diff
@@ -0,0 +1,13 @@
+doctype html
+html
+    head
+        title='Login page'
+        link(rel='stylesheet', href='style.css')
+    body
+        h1 Login Page
+        form(action='http://localhost:3000/auth/google', method='GET')
```

Comment:
zeby link dzialal na kazdym srodowisku, to nie dolaczaj adresu localhost, tylko uniwersalnie `/auth/google`